backend2/src/api/api_v1/endpoints/user_config.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any
+
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+
+from src.api import deps
+from src.db import models
+
+router = APIRouter()
+
+
+@router.put("/config-me/subscription-interval/{value-days}", status_code=201)
```

Comment:
Also, move the `value-days` param to the request body. Because having it in the URL suggests that the user can have many different value-days configs witch he don't.

For body params see https://github.com/tiangolo/full-stack-fastapi-couchbase/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/app/api/api_v1/endpoints/items.py#L84

Reply:
Didn't catch this one. Do you want these item description classes be added from the example? https://github.com/tiangolo/full-stack-fastapi-couchbase/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/backend/app/app/models/item.py

Reply:
I pushed the changes I meant. Please, take a look at my commit.